### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by Zhuxz111** [ImanRHT/QOCO#2]
<img width="871" alt="iShot_2023-12-05_17 51 09" src="https://github.com/ImanRHT/QOCO/assets/66059485/fd7ee536-e088-4839-8f64-9ddfa4eac0b0">
<img width="869" alt="iShot_2023-12-05_17 51 01" src="https://github.com/ImanRHT/QOCO/assets/66059485/eb4ed9e0-b943-4aaf-ae56-9993e7832a8a">
I think the formula (10) in this article is wrong, could you help me? Thanks.

**Comment by ImanRHT** [comment]
Yes, you’re absolutely right. We’ve revised formula accordingly. Thanks for pointing this out! 
The new version of the algorithm is now available and converges perfectly. [QECO](https://github.com/ImanRHT/QECO).

Please give it another try, and feel free to share any feedback or insights you may have.
@Zhuxz111
